conf/machine/t600.conf

```diff
@@ -1,8 +1,8 @@
 #@TYPE: Machine
 #@DESCRIPTION: Machine configuration for running T600 in 64-bit mode
 
-require ../../../meta-fsl-ppc/conf/machine/e6500-64b.inc
```

Comment:
I need layers to be able to float around to simply CI and this was failing when meta-fsl-ppc and meta where not exact 3 hops away.  Per bitbake manual:

> The directive is much like its equivalent in Make except that if the path specified on the include line is a relative path, BitBake locates the first file it can find within BBPATH.

And BBPATH is basically the same as BBLAYERS from build/conf/bblayers.conf... so adding the "../../.." doesn't really make much sense in this case.